packages/connectivity_plus/connectivity_plus/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.6.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.4-bin.zip
```

Comment:
Had to do it because of using 8.3.0 Gradle plugin, which didn't allow me to open just Android part of the plugin with 7.6.1 version of Gradle wrapper.

Reply:
I will open another PR where I will address such changes.